pages/common/aws-iam.md

```diff
@@ -0,0 +1,36 @@
+# aws iam
+
+> CLI for AWS IAM - web service to securely control access to AWS services and resources.
```

Comment:
```suggestion
> CLI for AWS IAM.
> Web service to securely control access to AWS services and resources.
```

Reply:
Change made. Thanks for the feedback.

Reply:
But you remove the `> Web service to securely control access to AWS services and resources.` text

Reply:
Oops sorry. I'll fix that when I'm back at my computer.